chore: Rename references to Storage Manager in docs to File Uploader (#5697)
aws-amplify/amplify-ui: docs/src/pages/[platform]/connected-components/storage/fileuploader/react.mdx

  UploadActionsExample,
} from './examples'

<Alert variation="info" heading="Storage Manager is renamed">

>   StorageManager component has been renamed `FileUploader` as of `aws-amplify/ui-react-storage` version `3.3.0`. Please upgrade to the new import path, as the `StorageManager` will be removed in a future major version.

Reviewer: ```suggestion
  StorageManager component has been renamed `FileUploader` as of `@aws-amplify/ui-react-storage` version `3.3.0`. Please upgrade to the new import path, as the `StorageManager` will be removed in a future major version.
```

Reviewer 2: ```suggestion
      Beginning with @aws-amplify/ui-react-storage@v3.3.0, the StorageManager
      component has been renamed to FileUploader. To migrate, upgrade to the
      latest version of @aws-amplify/ui-react-storage and replace StorageManager
      related imports with their corresponding FileUploader equivalents.
      
      StorageManager will continue to receive official support until the next
      major version release of @aws-amplify/ui-react-storage, at which point
      it will be replaced by FileUploader.
```

Reviewer: Let's go with Caleb's more detailed one

Author: Agreed. I just made a slight change of `_@aws-amplify/ui-react-storage@v3.3.0_` to `_@aws-amplify/ui-react-storage_ version 3.3.0`, as the original created unwanted links and the new pattern is in line with existing references on the page.

  UploadActionsExample,
} from './examples'

<Alert variation="warning" heading="Storage Manager is renamed">

>   StorageManager component has been renamed `FileUploader` as of `aws-amplify/ui-react-storage` version `3.3.0`. Please upgrade to the new import path, as the `StoraageManager` will be removed in a future major version.

Reviewer: ```suggestion
  StorageManager component has been renamed `FileUploader` as of `aws-amplify/ui-react-storage` version `3.3.0`. Please upgrade to the new import path, as the `StorageManager` will be removed in a future major version.
```

Author: Got it! Good eye

<AppDirectoryAlert />

> To use the FileUploader component import it into your React application with the included styles. 

Reviewer: We should add an informational alert here that StorageManager is now FileUploader

Author: Done. Should I still keep the "Storage Manager is deprecated" warning alert in the Deprecated props section?

</Example>

### Deprecated props

> <Alert variation="warning" heading="Storage Manager is deprecated">

Reviewer: ```suggestion
<Alert variation="warning" heading="Storage Manager renamed">
```

Reviewer: let's move this alert to the top of the page

Author: Done and done

>   This component used to be called Storage Manager. The Storage Manager component has been deprecated, and all functionality is retained by the File Uploader component.

Reviewer: ```suggestion
  StorageManager component has been renamed `FileUploader` as of version  `aws-amplify/ui-react-storage` version `3.3.0`. Please upgrade to the new import path, as the `StoraageManager` will be removed in a future major version.
```

Author: Done
